index.js

```diff
@@ -0,0 +1,122 @@
+
+//removes grid container from document flow
+document.getElementById('gridContainer').style.display="none"
+document.getElementById('gameTitle').style.display="none"
+
+
+//function to generate all game tiles
+document.querySelector('.form').addEventListener('submit', (e) => {
+    //stops form from submitting
+    e.preventDefault()
+    //Defining total grid size, number of lynx and misses from user input
+    let numLynx = parseInt(document.getElementById('numLynx').value)
+    let gridSize = parseInt(document.getElementById('gridSize').value)
+    let miss = gridSize - numLynx
+
+    //reset
+    document.getElementById('gridContainer').innerHTML = ''
+    document.getElementById('errorMessage').textContent = ''
+
+
+        let generateSquares = () => {
+            //create empty array
+            let squares = []
+            //For every hit we are putting a div into the array with data-hit 1
+            for (i = 0; i < numLynx; i++) {
+                squares.push('<div class="grid-item" data-hit="1"></div>')
+            }
+            //For every hit we are putting a div into the array with data-hit 0
+            for (i = 0; i < miss; i++) {
+                squares.push('<div class="grid-item" data-hit="0" ></div>')
+            }
+            //shuffle the array
+            let shuffle = (array) => array.map(a => ({
+                sort: Math.random(),
+                value: a
+            })).sort((a, b) => a.sort - b.sort).map(a => a.value)
+            let shuffledArr = shuffle(squares)
+            //adding each array element to html of container
+            shuffledArr.forEach((square) => {
+                document.getElementById('gridContainer').innerHTML += square
+            })
+        }
+
+    //validates form input
+    if(!validateInput(gridSize, numLynx)){
+        document.getElementById('errorMessage').textContent = errorMessage
+    }else {
+        //execute function
+        generateSquares()
+        turnCard()
+        //sets the grid container display to default
+        document.getElementById('gameTitle').style.display = ""
+        document.getElementById('gridContainer').style.display = ""
+        //scroll to top of game play area
+        document.querySelector('#gameTitle').scrollIntoView({
+            behavior: 'smooth'
+        })
+    }
+})
+
+
+// DOCBLOCK READ PLS
+// Function takes 2 integer inputs from form.
+//num1 is grid size.
+//num2 is the number of lynx.
+//Function checks if number is integer,
+// if positive
+// and if hitboxes (num2) does not exceed half of the grid size (num1).
+// global error message is populated upon error, dictated by the conditional, ready to be displayed
+//return true if no errors
+
+let errorMessage = ""
+
+function validateInput(num1, num2) {
```

Comment:
if you name the parameters better you wont need so much description. calling them `num1` and `num2` is not helpful